bundles/org.openhab.binding.intesis/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="intesis"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="intesisHome">
+		<label>IntesisHome WiFi Adapter</label>
+		<description>IntesisHome WiFi Adapter</description>
+
+		<channels>
+			<channel id="power" typeId="system.power"/>
+			<channel id="mode" typeId="mode"/>
+			<channel id="windspeed" typeId="windspeed"/>
+			<channel id="swingUpDown" typeId="swingUpDown"/>
+			<channel id="temperature" typeId="temperature"/>
+			<channel id="returnTemp" typeId="returnTemp"/>
+			<channel id="outdoorTemp" typeId="outdoorTemp"/>
+		</channels>
+
+		<config-description>
+			<parameter name="password" type="text" required="true">
+				<label>Password</label>
+				<description>Password for API access.</description>
+				<context>password</context>
+			</parameter>
+			<parameter name="ipAddress" type="text" required="true">
+				<label>IP Address</label>
+				<description>IP Address of the IntesisHome device.</description>
+				<context>network-address</context>
+			</parameter>
+		</config-description>
+
+	</thing-type>
+
+	<channel-type id="mode">
+		<item-type>Number</item-type>
+		<label>@text/channel-type.intesis.mode.label</label>
+		<description>@text/channel-type.intesis.mode.description</description>
+		<state>
+			<options>
+				<option value="0">Auto</option>
+				<option value="1">Heat</option>
+				<option value="2">Dry</option>
+				<option value="3">Fan</option>
+				<option value="4">Cool</option>
```

Comment:
If I see correctly, this doesn't match with the i18n file. As it is redundant, you could remove it in the i18n file.

Reply:
Translations are work in progress